src/api.v2/helpers/modelGuts.js

```diff
@@ -0,0 +1,66 @@
+const sqlClient = require('../../sql/sqlClient');
+
+// The guts of a model that uses Knexjs to store and retrieve data from a
```

Comment:
I understand that 'guts' is a Knex term, but is there a better name that is more common? The purpose of this module is to initialize an object with actions. How about `initializeModel` or `createModel`?

Reply:
I don't think it is a knex term, it might just be the word guts.

It adds some basic queries, so maybe we could call it `basicModelFunctions`

Reply:
renamed it to `generateBasicModelFunctions`